otter/tap/api.py

```diff
@@ -317,11 +318,10 @@ def setup_converger(parent, kz_client, dispatcher, interval, build_timeout,
         kz_client=kz_client,
         interval=interval,
         partitioner_path=CONVERGENCE_PARTITIONER_PATH,
-        time_boundary=15,  # time boundary
+        time_boundary=1,  # time boundary
```

Comment:
What is this? I see in a docstring "time to wait for partitioner to stabilize". Partly, I'm curious, and I want to make sure this is an intentional change as well.